std_msgs/Header header
nav_msgs/MapMetaData info
float32[] slope
